app/controllers/v0/claim_documents_controller.rb

```diff
@@ -69,5 +78,33 @@ def unlock_file(file, file_password)
       file.tempfile = tmpf
       file
     end
+
+    def stamped_pdf_valid?
+      extension = File.extname(@attachment&.file&.id)
+      allowed_types = PersistentAttachment::ALLOWED_DOCUMENT_TYPES
+      unless allowed_types.include?(extension)
+        raise Common::Exceptions::UnprocessableEntity.new(
+          detail: I18n.t('errors.messages.extension_allowlist_error', extension:, allowed_types:),
+          source: 'PersistentAttachment.unlock_file'
```

Comment:
this need to be updated?
also, add a separate check here for an empty file?

Reply:
Yes, I found from running our specs that trying to convert txt files into PDF caused issues within pdftk, hence the extra enforcement of file type here. i'll add a check for empty file here now